src/hbbft_rbc.erl

```diff
@@ -208,7 +208,7 @@ check_completion(Data = #rbc_data{n=N, f=F}, H) ->
     M = N - 2*F, %% Note: M = Threshold (specified in RBC protocol)
     K = 2*F,
 
-    Shards = maps:values(maps:get(H, Data#rbc_data.stripes, [])),
+    Shards = maps:values(maps:get(H, Data#rbc_data.stripes, #{})),
```

Comment:
`stripes` is a map, I think this got overlooked in one of the previous commits. It needed fixing but I'm unsure if it'll _actually_ help with anything tbh.